doc/design.md

```diff
@@ -15,6 +15,7 @@ This document's section structure is derived from the "[arc42](https://arc42.org
 This section introduces technical system constraints.
 
 ## Exasol Docker Container Requires Privileged Mode
+
```

Comment:
Don't add emtpy lines here. It should not affect the requirement tracing, but this is intentionally close together. Also it is an unnecessary change.